src/pages/guides/index.md

```diff
@@ -251,7 +251,8 @@ This feature is scheduled to be rolled out on 10th February, 2022
 For a more robust and reliable verification, Adobe I/O Events adds below security validations for events delivered to your webhook. 
 
 - sending additional field of recipient_client_id as part of your event payload
-- signing the event payloads digitally and sending those digital signatures as request headers to your webhook
+- signing all the event payloads digitally using a fixed public/private key pair generated by Adobe I/O Events and sending those digital signatures as request headers to your webhook
+- sending the relative paths of public keys, which are served from our fixed adobe domain adobeioevents.com so that no one else can use their arbitrary public/private key pairs, as webhook request headers.
 
 I/O Events generates two public-private key pairs and sign your event payload using digital signatures following the steps below
```

Comment:
> I/O Events generates two public-private key pairs

This gives the impression that the two keys are generated for every event. We should clarify that the two keys are  fixed (but might be rotated periodically / quarterly).

Reply:
Key rotation is something that is hidden from the consumer, should we mention that here? I think mentioning only `fixed` will suffice. wdyt?

Reply:
Yes, we need to inform our consumers that the keys will change with time. They should anticipate keys getting rotated and design their service accordingly. Maybe we can also add a separate section for rotation of keys.

Reply:
IMHO, adding a separate section for rotation will ask for more details in it like its implementation and all.. which wouldn't be necessary. irrespective of what rotation implementation we have asked consumer to cache for `<=24h`. We can however just mention that keys will change every quarter and in any security threat scenario. wdyt?

Reply:
okay